README.md

```diff
@@ -226,6 +226,62 @@ and React (using TSX files). Please use the React Generators for creation.
 * Push to the remote for `PRIVATE_PROD_DEPLOY` (ask Eric for the remote and access)
 * Push to heroku production  using `git commit production PRIVATE_PROD_DEPLOY:master` ( ask Eric for access to `production`)
 
+## (Mac Setup) Build for Production 
+# In order to get prod env set, you need to download Github CLI and Heroku CLI. 
+
+# Github CLI setup 
+```
+gh # to check if GH CLI has been downloaded 
+brew install gh  # if GH CLI has not been downloaded
+gh 
+gh auth
+gh auth login
+```
+# Answer the following questions: 
+—? What account do you want to log into? GitHub.com
+? What is your preferred protocol for Git operations? HTTPS
+? Authenticate Git with your GitHub credentials? Yes
+? How would you like to authenticate GitHub CLI? Login with a web browser
+—copy one time code, and enter into browser 
+
+
+# Heroku CLI setup 
+```
+brew tap heroku/brew && brew install heroku 
+heroku login 
+heroku git:remote —remote=production -a commitchange 
+git branch 
+git push production HEAD:master 
+```
+
+```
+git checkout supporter_level_goal
+./create_new_release.sh
+git push private HEAD:master
+git checkout PRIVATE_PROD_DEPLOY
+git merge PRIVATE_MASTER
+git push private HEAD:prod_deploy
+npm run build-all-production
+git add public
+git commit -m "<a build message>"
+git push private HEAD:prod_deploy
+git push production HEAD:master
+```
```

Comment:
Please swap this code block with the one below it. Additionally, please add a header `# Build for production` to this.